src/index.js

```diff
@@ -16,7 +16,11 @@ const unexpectedEnzyme = {
         return reactWrapper && reactWrapper instanceof ReactWrapper;
       },
       inspect: function(reactWrapper, depth, output) {
-        output.appendInspected(reactWrapper.getElement());
+        const inspected = reactWrapper.exists()
+          ? reactWrapper.getElement()
+          : reactWrapper.root().html();
```

Comment:
This is a bit weird, why would we show the root?

Reply:
Maybe we can do something fancy where we inspect the parent node in gray and the current element in colors. But let's do that later.

Reply:
It's the best we can do, I have been investigating, but `.parent()` does not seem to work either. Ideally we could recursively call `.parent()` until we found a valid node.

Reply:
👍